### ISSUE TITLE
 ItemStackRequest rejected ItemStackRequestPacket.Request Bug

### ISSUE THREAD

**Issue by LichLord91** [GeyserMC/Geyser#1932]
**Describe the bug**

I have this plugin that autosorts/autostacks/autorefills inventory and chest. For my bedrock players they occasionally get this error when the plugin tries to autosort something.

```
1 | [Mon, 15. Feb 2021 09:49:08 PST WARN] DEBUG: About to reject request made by messykan
2 | [Mon, 15. Feb 2021 09:49:08 PST WARN] Source: StackRequestSlotInfoData(container=INVENTORY, slot=33, stackNetworkId=1979) Result: false
3 | [Mon, 15. Feb 2021 09:49:08 PST WARN] Destination: StackRequestSlotInfoData(container=ARMOR, slot=0, stackNetworkId=0) Result: true
4 | [Mon, 15. Feb 2021 09:49:08 PST WARN] java.lang.Throwable: DEBUGGING: ItemStackRequest rejected ItemStackRequestPacket.Request(requestId=-783, actions=[PlaceStackRequestActionData(count=1, source=StackRequestSlotInfoData(container=INVENTORY, slot=33, stackNetworkId=1979), destination=StackRequestSlotInfoData(container=ARMOR, slot=0, stackNetworkId=0))], filterStrings=[])
```

Full debug log:

https://mclo.gs/fatMotB

Plugin:
https://dev.bukkit.org/projects/automatic-inventory

On version: 2.12.2

**To Reproduce**

1. Install Plugin
2. Login as Geyser/Floodgate Bedrock player
3.  Add some stackable/non-stackable items and check error debug logs

**Expected behavior**

No errors to occur

**Screenshots / Videos**

N/A

**Server Version and Plugins**

 "platform": "SPIGOT",
        "platformName": "Purpur",
        "platformVersion": "git-Purpur-978 (MC: 1.16.5)",
        "platformAPIVersion": "1.16.5-R0.1-SNAPSHOT",
        "onlineMode": true,

If you're running a multi-server instance, or using Geyser Standalone:

"version": "1.2.0-SNAPSHOT (git-server-inventory-ce33ca5)", Single instance Geyser-Spigot.jar

If this bug occurs on a server you do not control, please fill this in to the best of your knowledge.

**Geyser Dump**

https://dump.geysermc.org/OF6RNataJKN7khoOaj8zkaI2igzEBjXl

**Minecraft: Bedrock Edition Version**

v1.16.201 on Windows 10 and IOS and Xbox and PS4

**Additional Context**

**Comment by Camotoy** [comment]
Does this impact gameplay in any way?

**Comment by LichLord91** [comment]
Hey Camotoy, 

TLDR: I dont think it actually impacts play , more testing is needed on my side ill report back as soon as I can, I'm still currently unsure what scenarios this error gets triggered. It may be just doing its job in the end and might be not something needed to be fixed. Ill get back to you asap.

Sorry its been a hectic week. I will do some more testing on this. I believe the only impact i've seen thusfar is when doing like a /give sword or something that does not naturally stack and you get more than 1 in the same slot it'll have trouble processing that when u put it in the inventory and it tries to sort that "illegal" stack. There are other scenarios as well as my bedrock players get this all the time and they don't really have any access that would allow them to do those wierd stack items.  I do notice when you try to then distribute those items like having 64 potions stack the number kinda disappears from that stack while your lifting it but doesn't disappear completely or anything.

**Comment by LichLord91** [comment]
Still haven't taken a slice of time to test, will update when i can.

**Comment by RednedEpic** [comment]
is this still an issue?

**Comment by Chew** [comment]
Sadly no update was provided and no reports of this have occurred in the last 2-5 months, so we're gonna have to close this due to inactivity. Thank you for reporting!
